ddtrace/tracer.py

```diff
@@ -73,6 +73,12 @@
     " dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] "
 )
 if debug_mode and not hasHandlers(log) and call_basic_config:
+    deprecation(
+        name="ddtrace.tracer.logging.basicConfig",
+        message="ddtrace will no longer handle basic configuration for a logging system.\
+        `logging.basicConfig()` should be called in a user's application",
+        version="1.0.0",
+    )
```

Comment:
🤔 

I am not sure we can handle this with a deprecation actually. Since we do this by default we are basically going to add a warning log on every startup even when the user does nothing.

We might be better off just making the change on the `1.0-dev` branch and have it as an `upgrade` note and we can document in any migration guide.

We could maybe deprecate the `DD_CALL_BASIC_CONFIG` env variable? But not sure if it is fully necessary since most people use it to disable calling basic config, so should not have any impact on upgrade.

cc @Kyle-Verhoog what do you think? is there another deprecation approach I am not thinking of?

Reply:
I'll close this PR